scripts/copy-res.ts

```diff
@@ -124,7 +124,7 @@ function genLangFile(lang: string, dest: string): string {
 
     const json = JSON.stringify(translations, null, 4);
     const jsonBuffer = Buffer.from(json);
-    const digest = loaderUtils.getHashDigest(jsonBuffer, null, "hex", 7);
+    const digest = util.createHash("xxhash64").update(jsonBuffer).digest("hex").slice(0, 7);
```

Comment:
This is the equivalent of what `getHashDigest(jsonBuffer, null, "hex", 7)` used to produce. See https://github.com/webpack/loader-utils/blob/62e12f19c1b15779211da5f07b02250852838179/lib/getHashDigest.js#L63 and https://github.com/webpack/webpack/blob/87660921808566ef3b8796f8df61bd79fc026108/lib/util/createHash.js#L149 for details.